src/main/app/src/app/zaken/intake-afronden-dialog/intake-afronden-dialog.component.ts

```diff
@@ -28,21 +32,49 @@ export class IntakeAfrondenDialogComponent implements OnInit {
     loading = false;
     mailBeschikbaar = false;
     sendMailDefault = false;
+    initiatorEmail: string;
+    initiatorToevoegenIcon: ActionIcon = new ActionIcon('person', 'actie.initiator.email.toevoegen',
+        new Subject<void>());
+    loadingIcon = new ActionIcon('hourglass_empty', 'msg.loading', new Subject<void>());
     formGroup: FormGroup;
+    initiator: Observable<Klant>;
```

Comment:
Het ophalen van het email adres gaat snel Je hebt er dus geen loading indicator of zo voor nodig

Reply:
Eruit gehaald, was inderdaad niet meer nodig, het is nu snel genoeg